src/app/controllers/users/register.js

```diff
@@ -12,6 +12,7 @@ angular.module('flightNodeApp')
         function($scope, $log, messenger, roleProxy, authService, $location, userProxy) {
 
             $scope.loading = true;
+            $scope.showRoles = false;
```

Comment:
This is used to turn off the display of the Roles element in the Profile and Registration pages, where an end-user should not be allowed to set the Role.

The back-end also enforces the lowest level of role for a new registrant by default, and will enforce no change to roles for a profile update.